### ISSUE TITLE
Config definition for media path has changed in October

### ISSUE THREAD

**Issue by nizzac** [mplodowski/seomanager-plugin-public#16]
The SeoTags component attempts the get og image dimensions in the `setOgImageDimensions()` method.

It attempts to get the path for the media directory using

`config('cms.storage.media.path')`

but this has since been changed in OctoberCMS v2 to

`config('system.storage.media.path')`

This causes the `og_image_width` and `og_image_height` properties not to be set so throws an error on the frontend when an seo image is added.

**Comment by daftspunk** [comment]
To fix support for v2 (and keep support for v1).

1. Open **plugins\renatio\seomanager\components\SeoTags.php**
2. In method `setOgImageDimensions`
3. Update `$filePath` variable set to:
```
$filePath = base_path(config('system.storage.media.path', config('cms.storage.media.path')).$this->seoTag->og_image);
```

**Comment by mplodowski** [comment]
Thank you for this. Fixed in version 1.3.4.
